epictrack-web/src/components/work/WorkForm/index.tsx

```diff
@@ -318,7 +325,7 @@ export default function WorkForm({ ...props }) {
           ></ControlledSelectV2>
         </Grid>
         <Grid item xs={12}>
-          <ETFormLabel required>Title</ETFormLabel>
+          <ETFormLabel required>Title ({title?.length || 0}/150)</ETFormLabel>

```

Comment:
There's a reusable component for this I think, the ETFormLabelWithCharacterLimit

Reply:
updated with ETFormLabelWithChracterLimit